src/configs.test.ts

```diff
@@ -0,0 +1,98 @@
+import {readdirSync} from 'fs';
+import {join} from 'path';
+
+import {Config, Variant} from './types';
+
+describe('configs', () => {
+  const configsPath = join(__dirname, 'configs');
+  const configFileNames = readdirSync(configsPath);
+
+  describe.each(configFileNames)('%s', (configFileName) => {
+    // eslint-disable-next-line @typescript-eslint/no-var-requires
+    const config: Config = require(`configs/${configFileName}`).config;
+
+    describe.each(Object.entries(config))('property "%s"', (_, variants) => {
+      const variantsByName = variants.map(({name}, index) => [
+        name,
+        variants[index],
+      ]);
+      it.each(variantsByName)(
+        'variant "%s" uses a unique Figma name',
+        (_, variant) => {
+          try {
+            expect(variant).toUseUniqueFigmaName(config);
+          } catch (error) {
+            // Override stack trace to show problematic file path instead of test file
+            error.stack = `at src/configs/${configFileName}`;
+            throw error;
+          }
+        },
+      );
+    });
+  });
+});
+
+expect.extend({
+  toUseUniqueFigmaName(variant: Variant, config: Config) {
+    if (this.isNot === true) {
```

Comment:
I'm not familiar with Jest matchers, just curious if this condition is useful/needed?

Reply:
It's unlikely that anyone would stumble into this, but it will save somebody a headache if they ever try to use `expect(variant).not.toUseUniqueFigmaName()` (i.e. the inverted form of this assertion).

Reply:
I see, thank you for the context!